tools/test-manylinux.sh

```diff
@@ -55,8 +55,17 @@ function run_single_in_docker {
     hash -r # let go of previous 'pip'
 
     # Install modules
+
+    # TODO: revisit to avoid hardcoding dependencies
+    pip install "futures;python_version=='2.7'"
+    pip install "enum34;python_version=='2.7'"
+
     pip install confluent_kafka --no-index -f $wheelhouse
+
+    # Pytest relies on a new version of six; later versions of pip fail to remove older versions gracefully
+    # https://github.com/pypa/pip/issues/5247
     pip install pytest
+    pip install pytest --ignore-installed six
```

Comment:
do we need to install pytest twice?

Reply:
how embarrassing... that is likely what caused my issues with six to begin with